gcp_icone_translator.py

```diff
@@ -0,0 +1,142 @@
+from translator import icone_translator
```

Comment:
Likewise in here, we'll want a few simple unit tests showing success/fail cases

Reply:
This should have a test file, tests/gcp_icone_translator_test.py